quickwit/quickwit-cluster/src/cluster.rs

```diff
@@ -50,43 +50,46 @@ const GOSSIP_INTERVAL: Duration = if cfg!(any(test, feature = "testsuite")) {
 } else {
     Duration::from_secs(1)
 };
-const AVAILABLE_SERVICES_KEY: &str = "available_services";
+const ENABLED_SERVICES_KEY: &str = "enabled_services";
 
 /// Cluster member.
 #[derive(Clone, Debug, Eq, PartialEq)]
 pub struct ClusterMember {
-    /// An ID that makes a member unique. Chitchat node ID is built from
-    /// the concatenation `{node_unique_id}/{generation}`.
-    pub node_unique_id: String,
-    /// Timestamp (ms) when node starts.
-    pub generation: u64,
-    /// Gossip advertise address.
+    /// A unique node ID across the cluster.
+    /// The Chitchat node ID is the concatenation of the node ID and the start timestamp.
```

Comment:
Nitpick: you can keep the template format to show how it's built `{node_id}/{start_timestamp}`